lib/bolt/pal.rb

```diff
@@ -212,11 +226,11 @@ def with_bolt_executor(executor, inventory, pdb_client = nil, applicator = nil,
         apply_executor: applicator || Applicator.new(
           inventory,
           executor,
-          @modulepath,
+          @modulepath.bolt_compiler_modulepath,
```

Comment:
same question for both of these https://github.com/puppetlabs/bolt/pull/2252/files#r504079462